client/app/pages/dashboards/dashboard.js

```diff
@@ -134,6 +134,14 @@ function DashboardCtrl(
     this.globalParameters = this.dashboard.getParametersDefs();
   };
 
+  this.loadWidget = (widget, forceRefresh) => {
```

Comment:
We will need this in the public dashboard implementation too, right? :thinking:

Reply:
I guess Cypress is trying to tell me the same thing, but I'll check that, I think it's the last thing before finally opening this for review.